ingesters/SimpleRelay/config.go

```diff
@@ -43,7 +43,7 @@ type listener struct {
 	base
 	Tag_Name      string
 	Reader_Type   string
-	Keep_Priority bool // Leave the <nnn> priority value at the start of the log message
```

Comment:
won't this break existing configs if someone has set Keep-Priority=false?

Reply:
its undocumented and was never actually used.  but yeah, i'll leave it just in case

Reply:
addressed